README.md

```diff
@@ -24,12 +24,16 @@ with Flow().add(uses=my_image_tag):
 
 ## Contribute to This Repository
 
-### Create a New Executor
-
+### Installation
 ```bash
 git clone https://github.com/jina-ai/jina-hub.git
 
 pip install "jina[hub]"
+```
```

Comment:
I would put this in Prerequisites together with pip install jina[hub] and then keep the clone and run to create the executor.

Reply:
You mean like this?

```
# Prerequisites
pip install "jina[hub]"
Install Docker.

# Create a New Executor
git clone https://github.com/jina-ai/jina-hub.git

jina hub new --type pod
```

Reply:
Yes kind of